app/assets/javascripts/components/story/CollapsedStory/index.js

```diff
@@ -25,65 +29,187 @@ const storyClassName = (story, additionalClassname = '') => {
       'Story--estimated': !isStoryNotEstimated
     },
     additionalClassname
-  );
-};
+  )
+}
 
-export const CollapsedStory = ({ 
-  onToggle, 
-  story, 
-  updateCollapsedStory, 
-  project, 
-  className, 
-  title, 
-  highlight, 
-  isHighlightable 
-}) =>
-  <div
-    className={storyClassName(story, className)}
-    onClick={onToggle}
-    title={title}
-  >
-    <StoryPopover story={story}>
-      <div className='Story__icons-block'>
-        <StoryIcon storyType={story.storyType} />
-        <CollapsedStoryEstimate estimate={story.estimate} />
-        <StoryDescriptionIcon description={story.description} />
-      </div>
-    </StoryPopover>
-
-    <CollapsedStoryInfo story={story} />
-
-    <CollapsedStoryStateActions story={story}
-      onUpdate={(newAttributes) =>
-        updateCollapsedStory(story.id, project.id, newAttributes)}
-    />
-    {
-      isHighlightable && 
-      <CollapsedStoryFocusButon onClick={() => highlight(story.id)} />
+const type = {
+  story: 'STORY'
+}
+
+const calculatePosition = (aboveStory, bellowStory, storiesArray, index) => {
+  if (bellowStory === undefined) {
```

Comment:
When we use return inside a `if`, we don't need of `else`. :nail_care: 

```
const calculatePosition = (aboveStory, bellowStory, storiesArray, index) => {
  if (bellowStory === undefined) return (Number(storiesArray[index].position) + 1)
  if (aboveStory === undefined) return (Number(storiesArray[index].position) - 1);
  return (Number(bellowStory.position) + Number(aboveStory.position)) / 2;
}
```

Reply:
But we need to return the calculated position, there's 3 cases so we have 3 returns.

Reply:
Yes. But the return stop the function. If the conditional enter in some `if`, the `return` will `break` the follow steps.

Reply:
I think I don't get it. It should stop, we just want one of the returns